### PR TITLE
Change: Updates Organization Groups

### PR DESCRIPTION
**Pull request by CGoodwin90**
# General Checklist

- [ ] Affected Issues have been mentioned in the Closing issues section
- [ ] Documentation has been written/updated
- [ ] PR title is ready for inclusion in changelog

Adds Organization specific input & type for Organization Groups.

### CODE REVIEW

**Review by shreddedbacon — CHANGES_REQUESTED** [review]
The typedef for `getGroupProjectOrganizationAssociation` needs to be updated to use `AddGroupToOrganizationInput`

File: services/api/src/typeDefs.js [hunk 1]
```diff
@@ -1415,9 +1415,9 @@ const typeDefs = gql`
     """
     Get an organization by its ID
     """
-    organizationById(id: Int!): Organization
+    organizationById(organization: Int!): Organization
```
Comment (shreddedbacon) [inline services/api/src/typeDefs.js]: This must have crept in through the merge conflict

Comment (shreddedbacon) [reply services/api/src/typeDefs.js]: Need to go back to `id`

Comment (CGoodwin90) [reply services/api/src/typeDefs.js]: Oops, good catch! Fixed now

**Review by shreddedbacon — APPROVED** [review]
This all seems to work well. Again as organizations isn't released yet these changes make sense to do now

**Pull request merged**